tools/GenerateConcordKeys.cpp

```diff
@@ -322,20 +288,10 @@ int main(int argc, char** argv) {
       return -1;
     }
 
-    // Validate that all output files are valid before possibly wasting a
-    // significant ammount of time generating keys that cannot be output.
-    std::vector<std::ofstream> outputFiles;
-    for (uint16_t i = 0; i < n; ++i) {
-      outputFiles.push_back(std::ofstream(outputPrefix + std::to_string(i)));
-      if (!outputFiles.back().is_open()) {
-        std::cout << "Could not open output file " << outputPrefix << i << ".\n";
-        return -1;
-      }
-    }
-
     std::vector<std::pair<std::string, std::string>> rsaKeys;
-    for (uint16_t i = 0; i < n; ++i) {
+    for (uint16_t i = 0; i < n+ro; ++i) {
```

Comment:
This has the possibility to overlow, since we are adding to uint16_t values. In practice it's not an issue, but I think we should stop trying to optimize for configuration and just store everything as uint64_t in the future.